source/linux/Foundational_Components/Kernel/Kernel_Drivers/GPIO.rst

```diff
@@ -48,33 +47,186 @@ and interrupt generation.
 .. rubric:: User Layer
    :name: user-layer
 
-The GPIO driver can be used via  user space or by
-other drivers that may need to access pins as either input/outputs or
-interrupts. More information about this driver and GPIO usage in Linux
-can be found in the kernel documentation:
-
--  GPIO Interface:
-   Under Kernel directory Documentation/gpio/gpio.txt
--  GPIO Driver:
-   Under Kernel directory Documentation/gpio/driver.txt
-
+The GPIO driver can be used in user space via Linux CLI. This section provides examples, command-line-tools, and guidance for using GPIOs in user space.
 
 .. note ::
-   Since linux 4.8 the GPIO sysfs interface is deprecated. User space should use
+   Since linux 4.8, the GPIO sysfs interface is deprecated. User space should use
    the character device instead.
 
+# Example: Using GPIOs with the *libgpiod* library in user space to toggle a GPIO pin, which is connected to an LED on the SK-AM62B-P1 board:
+
+#. Locate *Table 2-23* under **2.6.16.3 User Test LEDs** in the *AM62x Starter Kit User's Guide (Rev. C)*: https://www.ti.com/lit/pdf/spruj40.
+
+- This example uses LED "LD11", AKA the **IO_EXP_TEST_LED**. It is connected to GPIO *U70.24(P27)*, AKA pin 27 of the *GPIO Port Expander* on the SK-AM62B-P1 board:
+
+.. Image:: /images/sk-am62b-p1-top.png
```

Comment:
Should at least define a width or height here so the image doesn't take up a large section of the screen.